docs/zh-CN/administrator-guide/variables.md

```diff
@@ -338,3 +338,6 @@ SET forward_to_master = concat('tr', 'u', 'e');
 
     是否将 bitmap 和 hll 类型的 count distinct 查询重写为 bitmap_union_count 和 hll_union_agg 。
 
+* `prefer_broadcast_join`
```

Comment:
How about name it to "prefer_join_method", and value is "broadcast" or "shuffle"